src/main/java/com/yahoo/sketches/theta/ContexTheta.java

```diff
@@ -0,0 +1,56 @@
+package com.yahoo.sketches.theta;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class ContexTheta {
```

Comment:
The context should be a sub-class of UpdateSketch and should implements its API so it can replace the sketch in the application level